tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+---
+
+# Remote Staking via XCM
+
+![Banner Image](/images/tutorials/remote-staking-via-xcm/remote-staking-via-xcm-banner.png)
+
+## Introduction {: #introduction } 
+In this tutorial, we’ll stake DEV tokens remotely by sending XCM instructions from an account on the Moonbase Relay Chain (equivalent to the Polkadot Relay Chain). This tutorial assumes a basic familiarity with [XCM](/builders/xcm/overview/){target=_blank} and [Remote Execution via XCM](builders/xcm/xcm-transactor/){target=_blank}. You don’t have to be an expert on these topics but you may find it helpful to have some XCM knowledge as background. 
+
+There are actually two possible approaches for staking on Moonbeam remotely via XCM. We could send a remote EVM call that calls the [staking precompile](builders/pallets-precompiles/precompiles/staking/){target=_blank}, or we could use XCM to call the [parachainStaking pallet](builders/pallets-precompiles/pallets/staking/){target=_blank} directly without interacting with the EVM. For this tutorial, we’ll be taking the latter approach and interacting with the parachainStaking pallet directly. 
+
+**Note that there are still limitations in what you can remotely execute through XCM messages.** In addition, **developers must understand that sending incorrect XCM messages can result in the loss of funds.** Consequently, it is essential to test XCM features on a TestNet before moving to a production environment.
+
+## Prerequisites
+For development purposes we’ll be taking these steps on Moonbase Alpha and Moonbase Relay using TestNet funds. As a prerequisite, you should have DEV tokens which you can acquire from the [Moonbase Alpha faucet](https://apps.moonbeam.network/moonbase-alpha/faucet/){target=_blank} as well as some UNIT, the native token of the Moonbase Relay Chain. You can swap some DEV for xcUNIT here on [Moonbeam Swap](https://moonbeam-swap.netlify.app/#/swap){target=_blank} and then withdraw them to [your account on the Moonbase Relay Chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank} using [apps.moonbeam.network](https://apps.moonbeam.network/moonbase-alpha/){target=_blank}. 
+
+Copy the account of your existing or newly created account on the [Moonbase Relay Chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank}. You're going to need it to calculate the corresponding derivative account. A derivative account is a special type of account that’s keyless (the private key is unknown). Transactions from a derivative account can be initiated only via valid XCM instructions from the corresponding account on the relay chain. In other words, you are the only one who can initiate transactions on your derivative account - and if you lose access to your Moonbase Relay account, you’ll also lose access to your derivative account. 
```

Comment:
```suggestion
Copy the account of your existing or newly created account on the [Moonbase Relay Chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank}. You're going to need it to calculate the corresponding multilocation derivative account. A multilocation derivative account is a special type of account that’s keyless (the private key is unknown). Transactions from a multilocation derivative account can be initiated only via valid XCM instructions from the corresponding account on the relay chain. In other words, you are the only one who can initiate transactions on your multilocation derivative account - and if you lose access to your Moonbase Relay account, you’ll also lose access to your  multilocation derivative account. 
```

Reply:
We refer to it as multilocation derivative account. Fixing this I realized that the word _multilocation derivative_ account is repeated a lot here, so maybe rephrase it a bit

Reply:
Please fix this in the rest of the document (`derivative` for `multilocation derivative`)